### PR TITLE
Add Mattheus, Sean, Michael, and William to AUTHORS.md

### PR DESCRIPTION
**Pull request by U8NWXD**
Add recent contributors to the `AUTHORS.md` file

-----------------------------------------------------------------------

By creating this pull request, I agree to the Contributor License
Agreement, which is available in `CLA.md` at the top level of this
repository.

### CODE REVIEW

**Comment by U8NWXD** [comment]
@matt-wolff @thalassemia @WilliamIX can you check that you are listed in the authors file the way you want? Please approve the PR if you're happy with it

**Review by thalassemia — APPROVED** [review]
My part looks good. Thanks!

**Comment by eagmon** [comment]
Did you ask Michael?

**Review by matt-wolff — APPROVED** [review]
Looks good!

**Review by WilliamIX — APPROVED** [review]

File: AUTHORS.md [hunk 1]
```diff
@@ -16,3 +16,8 @@ Vivarium Core was built by:
   * Ryan Spangler: `ryan.spangler AT gmail DOT com`
   * [Chris Skalnik](https://u8nwxd.github.io)
   * Jerry Morrison: `j.erry.morrison AT gmail DOT com`
+  * Mattheus Wolff: `mbwolff AT stanford DOT edu`
+  * Sean Cheah: `seancheah AT ucla DOT edu`
+  * Michael Yang (@Robotato): `michaelyang42 AT gmail DOT com`
```
Comment (Robotato) [inline AUTHORS.md:21]: Looks good to me, thanks!

**Review by Robotato — APPROVED** [review]

**Pull request merged**